### ISSUE TITLE
Possibility of xmovie util

### ISSUE THREAD

**Issue by zssherman** [openradar/xradar#88]
Hello!

So we were discussing for ACT that a movie creator for radar data/images would be useful, but wouldn't really fall into the scope of ACT. With xradar utilizing xarray, which is what xmovie does:
https://xmovie.readthedocs.io/en/latest/examples/quickstart.html

Would this be something we'd want in xradar? Or an example of?

**Comment by kmuehlbauer** [comment]
An example notebook, how to utilize xmovie with xradar would be really great.

**Comment by mgrover1** [comment]
Yeah, I agree, having an example showing how to use this with xradar would be great!

**Comment by zssherman** [comment]
The notebooks in xradar run during testing correct? Trying to think how to have xmovie, I assume optional dependency, or something but wasn't sure how to handle it for the notebook example

**Comment by kmuehlbauer** [comment]
Yes, they are running during testing. Need to add xmovie to ci/unittests.yml.

They are also run on readthedocs.  Need to add xmovie to environment.yml.

**Comment by zssherman** [comment]
Gotcha sounds good!

**Comment by zssherman** [comment]
Made the mistake, xmovie hasn't been updated yet for datatrees, so closing for now.

**Comment by zssherman** [comment]
After discussion with @mgrover1, will leave open, until Max merges a multi sweep example.

**Comment by kmuehlbauer** [comment]
Yes, the xmovie guide might just be appended to that notebook. :+1:

**Comment by kmuehlbauer** [comment]
There is basic matplotlib animation in https://docs.openradarscience.org/projects/xradar/en/stable/notebooks/multiple-sweeps-into-volume-scan.html.

xmovie seems unmaintained https://github.com/jbusecke/xmovie/issues/163. I've had nice experience with https://github.com/ahuang11/streamjoy lately.

Closing this as resolved.
